src/protagonist/CleanupHandler/AssetDeletedHandler.cs

```diff
@@ -102,26 +102,18 @@ private async Task<bool> InvalidateContentDeliveryNetwork(Asset asset, string cu
             $"{customerName}/{asset.Id.Space}/{asset.Id.Asset}"
         };
         
-        if (asset.DeliveryChannels.IsNullOrEmpty())
-        {
-            logger.LogDebug("Received message body with no 'deliveryChannels' property. {@Request}", 
-                asset);
-        }
-        else
+        if (!asset.ImageDeliveryChannels.IsNullOrEmpty())

```

Comment:
Did you run the code end to end as it's called from API? I don't think this property will be populated.